src/redux/types/actions/index.ts

```diff
@@ -112,8 +114,13 @@ export type ActionTypeString = AllActions['type'];
 
 export type TakeFilter = (action: AllActions) => boolean;
 
+// To be replaced with MetaWithNavigate and deleted
 export type MetaWithHistory<M> = {
   history?: {
     push: <A>(route: A) => void;
   };
 } & M;
+
+export type MetaWithNavigate<M> = {
```

Comment:
The history object is no longer exposed by `react-router`, so all sagas with redirect will need to use this instead